about.html

```diff
@@ -65,6 +65,33 @@ <h2>Where are we now?</h2>
         <div class="about-faqs row pt-2">
           <h2>FAQs</h2>
           <div class="accordion" id="accordionExample">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+                  Will the McGuinness Boulevard redesign redirect traffic onto side streets?
+                </button>
+              </h2>
+              <div id="collapseTwo" class="accordion-collapse collapse" data-bs-parent="#accordionExample">
+                <div class="accordion-body">
+                  <p>The McGuinness Boulevard redesign is carefully planned to improve traffic flow and safety without causing significant disruption to side streets. One of the common concerns when a road is redesigned is that traffic will be diverted to smaller side streets, but this is not necessarily the case.</p>
+                  <p></p>Induced demand occurs when an expansion of road capacity leads to more people choosing to drive, which can often negate any improvements in traffic flow. Conversely, when road capacity is reduced, as in the McGuinness Boulevard redesign, some of the traffic may simply evaporate. This is known as traffic evaporation. People may choose alternate routes, switch to public transportation, or change the times they travel.</p>
```

Comment:
Any chance we could link to something for "traffic evaporation"? Maybe [this](https://nacto.org/docs/usdg/disappearing_traffic_cairns.pdf)?